src/test/rustdoc/issue-12834.rs

```diff
@@ -15,7 +15,7 @@
 // @has issue_12834/fn.foo.html
 // @has - //pre 'a + b '
 
-/// ```
+/// ```text
```

Comment:
Certainly but I can't remember which right now. I'll check again once the other issue is solved.